source/chapters/appendix/mixxx_controls.rst

```diff
@@ -780,7 +780,9 @@ Any control listed above for :mixxx:cogroupref:`[ChannelN]` will work for a samp
                    [PreviewDeckN],bpm_tap
                    [SamplerN],bpm_tap
 
-   When tapped repeatedly, adjusts the :term:`tempo` of the channel to match the tapped :term:`BPM`.
+   When tapped repeatedly, adjusts the :term:`tempo` of the track on the deck to match the tapped :term:`BPM`.
+
+   .. note:: If you want to change the :term:`rate` of the channel (instead of the :term:`BPM` of the file), use `script.bpm.tapButton(deck) <https://github.com/mixxxdj/mixxx/wiki/midi%20scripting#Helper-functions>`_ in your controller mapping instead.
```

Comment:
I think, to avoid confusion in the first place, we should use 'BPM' here, and maybe even mention the displayed beatgrid as well. The note could then only mention the tempo function.

```suggestion
   When tapped repeatedly, adjusts the :term:`BPM` of the track on the deck (not the tempo slider!) to match the taps.

   .. note:: If you want to change the :term:`rate` of the deck use `script.bpm.tapButton(deck) <https://github.com/mixxxdj/mixxx/wiki/midi%20scripting#Helper-functions>`_ in your controller mapping instead.
```